gst/nnstreamer/tensor_source/tensor_src_iio.c

```diff
@@ -1603,12 +1612,45 @@ gst_tensor_src_iio_get_times (GstBaseSrc * basesrc, GstBuffer * buffer,
 
 /**
  * @brief create a buffer with requested size and offset
+ * @note offset ignored as there is no header
+ * @note size ignored as done in pushsrc
  */
 static GstFlowReturn
 gst_tensor_src_iio_create (GstBaseSrc * src, guint64 offset,
     guint size, GstBuffer ** buffer)
 {
-  /** FIXME: fill this function */
+  GstTensorSrcIIO *self;
+  GstBuffer *buf;
+  GstMemory *mem;
+  GstMapInfo info;
+  guint buffer_size;
+
+  self = GST_TENSOR_SRC_IIO_CAST (src);
+  buffer_size = gst_tensor_info_get_size (&self->tensors_config->info.info[0]);
+
+  mem = gst_allocator_alloc (NULL, buffer_size, NULL);
+  if (!gst_memory_map (mem, &info, GST_MAP_WRITE)) {
+    goto error_mem_free;
+  }
+  info.size = size;
+  gst_memory_unmap (mem, &info);
+
+  buf = gst_buffer_new ();
+  gst_buffer_append_memory (buf, mem);
+
+  if (gst_tensor_src_iio_fill (src, offset, buffer_size, buf) != GST_FLOW_OK) {
```

Comment:
[Question]
gst_tensor_src_iio_fill() is vmethod of base-src. Is this called when implementing _create() function?

Reply:
Yes, implementation of _create() in base-src calls gst_tensor_src_iio_fill().

Reply:
Then I think this is unnecessary, base-src calls_create() and _fill()

Reply:
Please correct me if i am wrong - as per my understanding, as we have overridden the base class _create(), we will have to call _fill() from _create().

Reply:
I misunderstand your comment. (I understood that basr-src calls both _create() and _fill() , then we don't need to call _fill() in the create() function )
```When implements _create(), then base-src never call the vmethod _fill()``` is it correct?

Reply:
> When implements _create(), then base-src never call the vmethod _fill()
Yes. _fill() is only called from vmethod _create() of base-src, which we override.

Reply:
I understand, thanks for the detailed explanation.